codeSnippets/snippets/gson/src/test/kotlin/com/example/GsonAppTest.kt

```diff
@@ -55,4 +60,28 @@ class GsonAppTest {
             response.bodyAsText()
         )
     }
+
+    @Test
+    fun testGsonDateFormat() = testApplication {
+        application {
+            main()
+        }
+
+        val client = createClient {
+            install(ContentNegotiation) {
+                gson {
+                    registerTypeAdapter(LocalDate::class.java, LocalDateAdapter())
+                    setDateFormat(DateFormat.LONG, DateFormat.SHORT)
+                    setPrettyPrinting()
+                }
+            }
+        }
+
+        val response = client.get("/v1")
+        val responseModel = response.body<Model>()
+        assertEquals(HttpStatusCode.OK, response.status)
+        assertEquals(model, responseModel)
+
```

Comment:
The two blank lines here can be removed.

Reply:
done :+1: